_posts/2021-12-31-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,348 @@
+was published. We reached out to multiple projects to update compatibility to
+Trino.
+
+* [DBeaver](https://github.com/dbeaver/dbeaver/pull/10925)
+* [QueryBook](https://github.com/pinterest/querybook/issues/509)
+* [Homebrew](https://github.com/Homebrew/homebrew-core/pull/83185)
+* [dbt](https://github.com/dbt-labs/dbt-presto/issues/39)
+* [sqlalchemy](https://github.com/dungdm93/sqlalchemy-trino/issues/20)
+* [sqlpad](https://github.com/sqlpad/sqlpad/pull/974)
+* [Apache Superset](https://github.com/apache/superset/pull/13105)
+* [Redash](https://github.com/getredash/redash/pull/5411)
+* [Awesome Java](https://github.com/akullpp/awesome-java/pull/917)
+* [Awesome For Beginners](https://github.com/MunGell/awesome-for-beginners/pull/933)
+* [Airflow](https://github.com/apache/airflow/pull/15187)
+* [trino-gateway](https://github.com/lyft/presto-gateway/issues/134)
+* [Metabase](https://github.com/metabase/metabase/issues/17532)
+* and so much more...
+
+Not only was the community excited and supportive about the brand change, but
+particularly they were all loving the new mascot. Our adorable bunny was soon 
+after [named Commander Bun Bun by the community](/episodes/10.html).
+
+<p align="center">
+ <a href="https://twitter.com/jtannady/status/1346888143459545092" target="_blank">
+   <img align="center" width="50%" src="/assets/blog/2021-review/cbb.png"/>
+ </a>
+</p>
+
+Immediately after the announcement, the [Trino GitHub repository](https://github.com/trinodb/trino)
+started trending in number of stargazers. Up until this point, the PrestoSQL
+GitHub repository had only amassed 1,600 stargazers in the two years since it 
+had split from the PrestoDB repository. However, within four months after the 
+renaming, the number of stargazers had doubled. GitHub stars, issues, pull 
+requests and commits started growing at a new trajectory.
+
+<p align="center">
+ <a href="https://twitter.com/bitsondatadev/status/1344028682126565381" target="_blank">
+   <img align="center" width="50%" src="/assets/blog/2021-review/trending.png"/>
+ </a>
+</p>
+
+## Community growth
+
+While we are on the subject, lets talk about the health of the Trino community.
+As mentioned the GitHub stargazers trajectory skyrocketed. At the time of 
+writing, we just hit 4,600 stargazers on GitHub. This means, we have added over
+3,000 stargazers in the last year, a 187% increase.
+
+Aside from GitHub, we have seen a 47% growth of [the Trino Twitter](https://twitter.com/trinodb) 
+followers this year. [The Trino Slack community](https://trino.io/slack.html),
+where a large amount of troubleshooting and development discussions occur, saw a
+75% growth nearing 6,000 members. Finally, [the Trino YouTube channel](https://www.youtube.com/c/TrinoDB)
+has seen an impressive 280% growth in subscribers as the [Trino Community Broadcast](/broadcast/)
+has grown in the number of helpful videos for community members.
```

Comment:
> has grown in the number of helpful videos for community members.

This sounds a bit meh. Maybe talk about the number of episodes we published on various topics (with some examples)

Reply:
Agreed.

A lot of the increase on this channel was due to the [Trino Community Broadcast](/broadcast/), 
that brought users and contributors from the community to cover 23 episodes
about the following topics:

* Trino ecosystem (dbt, Amundsen, Debezium, Superset) - 7
* Trino project (Renaming Trino, Intro to Trino, )  - 4
* Trino connectors (Iceberg, Druid, Pinot) - 4
* Trino internals (Distributed Hash-Joins, Dynamic Filtering, Views) - 4
* Trino on Kubernetes (Trinetes series) - 2
* Trino users (LinkedIn, Resurface) - 2